apps/admin-x-activitypub/src/views/Profile/Profile-RR.tsx

```
@@ -0,0 +1,496 @@
+
+    return (
+        <div>
+            {
+                hasNextPage === false && posts.length === 0 ? (
+                    <NoValueLabel icon='pen'>
+                        {handle} has not posted anything yet
+                    </NoValueLabel>
+                ) : (
+                    <>
+                        {posts.map((post, index) => (
+                            <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)